ceed-qfunction.c

```diff
@@ -92,6 +92,7 @@ int CeedQFunctionAddInput(CeedQFunction qf, const char *fieldname,
 
 int CeedQFunctionAddOutput(CeedQFunction qf, const char *fieldname,
                           CeedInt ncomp, CeedEvalMode emode) {
+  if (emode == CEED_EVAL_WEIGHT) return CeedError(qf->ceed, 1, "Cannot create qfunction output with CEED_EVAL_WEIGHT");
```

Comment:
Looks like this commit only does the first part of the commit message.

Reply:
Updated